docs/source/convex-hull.rst

```diff
@@ -0,0 +1,46 @@
+Convex Hull
+==========
```

Comment:
The line with '=' (equals) should match the size of the title
Example:
Title Length
=========